messenger/http_transporter_test.go

```diff
@@ -17,11 +19,8 @@ import (
 )
 
 func TestTransporterNew(t *testing.T) {
-	id, err := upid.Parse(fmt.Sprintf("mesos1@localhost:%d", getNewPort()))
-	assert.NoError(t, err)
-	trans := NewHTTPTransporter(id, nil)
+	trans := NewHTTPTransporter(upid.UPID{ID: "mesos1", Host: "localhost"}, nil)
```

Comment:
Parsing the UPID was a lazy approach to UPID creation. There's no
compelling reason to do it that way.

On Thu, Sep 24, 2015 at 5:40 AM, Tomás Senart <notifications@github.com>
wrote:

> In messenger/http_transporter_test.go
> <https://github.com/mesos/mesos-go/pull/189#discussion_r40301368>:
>
> > @@ -17,11 +19,8 @@ import (
> >  )
> >
> >  func TestTransporterNew(t *testing.T) {
> > -	id, err := upid.Parse(fmt.Sprintf("mesos1@localhost:%d", getNewPort()))
> > -	assert.NoError(t, err)
> > -	trans := NewHTTPTransporter(id, nil)
> > +	trans := NewHTTPTransporter(upid.UPID{ID: "mesos1", Host: "localhost"}, nil)
>
> Why aren't we passing the id in like we did before?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesos/mesos-go/pull/189/files#r40301368>.
>